std/stdio.d

```diff
@@ -832,21 +832,19 @@ $(D rawRead) always reads in binary mode on Windows.
         return buffer;
     }
 
+    ///
     unittest
     {
-        static import std.file;
+        import std.file;
 
         auto deleteme = testFilename();
```

Comment:
No, I think you missed my point. `std.file.deleteme` is not supposed to be public; it's supposed to be used only internally by Phobos unittests. As such, I think it's a very bad idea to include it in public-facing documentation.  Moreover, `deleteme` is a *horrible* name that should not be exposed to the user. If we ever decide to export the functionality, we really should give it a better name.